user-guide/network/nictag.rst

```diff
@@ -46,9 +59,43 @@ Adding a NIC Tag
         external       -                  aggr0
         admin          -                  aggr0
 
+Adding etherstub NIC tag
+------------------------
+
+Etherstubs are created with the command shown below.. Please make sure that etherstub name ends with a number, otherwise you will end up with an error.
+Etherstubs created this way will be written to ``/usbkey/config``, and this makes them persistent over the reboots.
+
+.. code-block:: bash
+
+        [root@node01 ~] nictagadm add -l stub0
+        [root@node01 ~] nictagadm list
+        NAME           MACADDRESS         LINK           TYPE
+        stub0          -                  -              etherstub
+
+Adding overlay NIC tag
+----------------------
+
+As opposed to other NIC tag types, overlays are not created using ``nictagadm`` command. They are created by adding ``/var/run/smartdc/networking/overlay_rules.json`` and ``/var/run/smartdc/networking/my_overlay1.json`` files.
+
+``overlay_rules.json`` provides information to system about properties of the overlays. Here is the example of how this file might look like:
+
+.. code-block:: bash
+
+        [root@node01 ~] cat /var/run/smartdc/networking/overlay_rules.json
+        {
+        "my_overlay1": "-e vxlan -p vxlan/listen_ip=192.168.100.100,vxlan/listen_port=4790 -s files -p files/config=/var/run/smartdc/networking/my_overlay1.json -p mtu=1400",
+        "my_overlay2": "-e vxlan -p vxlan/listen_ip=192.168.200.200,vxlan/listen_port=4791 -s files -p files/config=/var/run/smartdc/networking/my_overlay2.json -p mtu=1400",
+        "my_overlay3": "-e vxlan -p vxlan/listen_ip=0.0.0.0,vxlan/listen_port=4790 -s files -p files/config=/var/run/smartdc/networking/my_overlay3.json -p mtu=1400"
+        }
+
+``my_overlay1.json`` provides ARP information, so that VMs connected to *my_overlay1* overlay network can find each other. The structure of this file is out of the scope of this section.
+
+
```

Comment:
Does it make sense documenting `files` overlay? The names of the ARP files will probably change in the future.
Or you can document also adding a `direct` overlay that will work out of the box now.

Reply:
I will remove this part.